numpy/core/include/numpy/experimental_dtype_api.h

```diff
@@ -321,6 +327,28 @@ typedef int (PyArrayMethod_StridedLoop)(PyArrayMethod_Context *context,
         NpyAuxData *transferdata);
 
 
+/**
+ * Query an ArrayMethod for the initial value for use in reduction.
+ *
+ * @param context The arraymethod context, mainly to access the descriptors.
+ * @param reduction_is_empty Whether the reduction is empty. When it is, the
+ *     default value for the identity might differ, for example:
+ *     - `0.0` is the default for `sum([])`.  But `-0.0` is the correct
+ *       identity otherwise as it preserves the sign for `sum([-0.0])`.
+ *     - We use no identity for object, but return the default of `0` and `1`
+         for the empty `sum([], dtype=object)` and `prod([], dtype=object)`.
+ *     - `-inf` or `INT_MIN` for `max` is an identity, but at least `INT_MIN`
+ *       not a good *default* when there are no items.
+ * @param initial Pointer to initial data to be filled (if possible)
+ *
+ * @returns -1, 0, or 1 indicating error, no initial value, and initial being
+ *     successfully filled.  Errors must not be given where 0 is correct, NumPy
+ *     may call this even when not strictly necessary.
+ */
+#define NPY_METH_get_reduction_initial 3
```

Comment:
Is this here on purpose or can it move up to the other defines?

Reply:
I have it grouped the corresponding typedef and there are 2 more of these that are "stray" in that sense.  I am happy to put them together but because of that didn't.